catalyst/data/reader.py

```diff
@@ -15,14 +15,14 @@ class ReaderSpec:
     All inherited classes have to implement `__call__`.
     """
 
-    def __init__(self, input_key: str, output_key: str):
+    def __init__(self, input_key: str, output_key: Optional[str] = None):
         """
         Args:
             input_key (str): input key to use from annotation dict
-            output_key (str): output key to use to store the result
+            output_key (Optional[str]): output key to use to store the result
```

Comment:
could you please add docs about `self.output_key = output_key or input_key` ? just to make users prepared